docs/dcp2_schema_evolution_specs.rst

```diff
@@ -0,0 +1,158 @@
+
+.. role:: raw-html(raw)
+   :format: html
+.. |nn| replace:: :raw-html:`<blockquote>`
+.. |ne| replace:: :raw-html:`</blockquote>`
+
+==============================
+DCP2 Metadata Schema Evolution
+==============================
+
+.. contents::
+
+Overview
+========
+
+The HCA metadata standard is developed in a transparent and open manner so that the whole HCA community can participate
+in the process. It follows a set of `design principles`_ that serve as a foundation for the metadata standard in order
+to ensure it meets the needs of HCA data contributors, consumers and DCP downstream components.
+
+The metadata schemas are versioned following semantic `versioning rules`_ that account for backwards compatibility.
+However, compatibility of the schemas does not account for impact on the schema consumers, especially if different
+versions of the same schemas must be maintained, indexed, displayed and/or interacted with in any form.
+
+The metadata is updated through PRs in the `metadata-schema repository`_. The specific details can be found in the
+documentation in the repository, but the overall flow can be described as follows:
```

Comment:
```suggestion
```

Reply:
It's ok to link to other texts but it's not ok to "conceptually include" texts via link that you think are normative. A spec needs to be self-contained as much as possible.